pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -1097,7 +1113,7 @@ def do_test(self, repository=None, remote=None):
         monitor_task(sync_response.task)
         return self.repo_api.read(repo.pulp_href), self.remote_api.read(remote.pulp_href)
 
-    def sync(self, repository=None, remote=None, optimize=True, mirror=False):
+    def sync(self, repository=None, remote=None, optimize=True, mirror=False, skip_types=[]):
```

Comment:
Using mutable collections in function signatures is somewhat dangerous.  It it's probably fine since it's in a task and the state gets thrown away quickly, but I'd still rather not.

(the danger is that `skip_types` is a single list that will be shared between all invocations of the function, not recreated, so if anything modifies `skip_types` in place it will be used by subsequent function calls even if the caller didn't provide `skip_types`).

How about 

```python
skip_types=None
...
...
if skip_types is None:
    skip_types = []
```

Reply:
FWIW, it's in the test but I can change it.

Reply:
pushed